### ISSUE TITLE
Revisit the Vuex store state for `fetching` in the Order Manager

### ISSUE THREAD

**Issue by jrgriffiniii** [pulibrary/figgy#3402]
I introduced some changes in https://github.com/pulibrary/figgy/pull/3401 which rely upon `window` for supporting Order Manager operations, but these should really be abandoned in favor of something which properly operates on the Vuex Store: https://vuex.vuejs.org/guide/state.html

**Comment by sdellis** [comment]
Just wanted to chime in that this is the proper way to fix this bug. The fetching variable in the component should reference the vuex state variable, not a local component variable.

**Comment by tpendragon** [comment]
@kelea99 This was marked as high priority on your document. Was there a particular reason?

**Comment by kelea99** [comment]
@tpendragon looking now... I saw the bug label/ordermanager here while flying through tickets. but see that it was removed.  What I understood from James's comment above: he introduced bits (that should be abandoned in favor of something else?) that depend upon something that is needed for Order Manager operations. If everything is fine as is and it won't hinder or mess with Order Manager? It doesn't have to be high priority. Y'all know oodles more and much better than I, so I default to your good judgement.
